README.md

```diff
@@ -27,3 +27,12 @@ Then you can add bridge as any other dependency.
 
 `    implementation("bridge:bridge-0.0.0")`
 
+# Test Coverage
+The test coverage reports are created using [JaCoCo](https://www.eclemma.org/jacoco/).
+
+Test coverage can be generated either
+` $./gradlew build
+` $./gradlew test jacocoTestReport
+
+JaCoCo generates HTML reports which canb be found at "./bridge/build/reports/jacoco/test/html"
```

Comment:
Update the path of the report according the ./gradlew task executed please so that others can guide themselves.

Reply:
that's the path from project root

Reply:
Are you sure about that? I've removed the older build and generated a new one with`$ ./gradlew test jacocoTestReport` and the path is there.

Reply:
I agree with you with you on this one `$ ./gradlew test jacocoTestReport`, but on `$ ./gradlew build` task the path is different.
![image](https://github.com/Pending-Name-21/frontend-library/assets/73414963/e038d9dd-15f2-4a11-abd7-871fa41c907f)

My suggestion is to update something like this:

Test coverage can be generated either
` $./gradlew build` 
`$./gradlew test jacocoTestReport`

Using build task the path is "./bridge/build/reports/tests/test/index.html"
JaCoCo generates HTML reports which **canb** be found at "./bridge/build/reports/jacoco/test/html"

Also, update the canb word to can please.

Reply:
oke